.devcontainer/devcontainer.json

```
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+  "name": "Ruby 2.5 on Debian",
+  "image": "ruby:2.5-slim-buster", // Specify the Ruby 2.5 image
```

How about using non slim version for  development?

```suggestion
  "image": "ruby:2.5-buster", // Specify the Ruby 2.5 image
```